O/onnxruntime/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "onnxruntime"
+version = v"1.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/microsoft/onnxruntime.git", "0d9030e79888d1d5828730b254fedc53c7b640c1")
+]
+
+# Bash recipe for building across all platforms
+script = """
+onnxruntime_version=$version
+""" * raw"""
+cd $WORKSPACE/srcdir
+
+if [[ $target == *-w64-mingw32 ]]; then
+    if [[ $target == x86_64-w64-mingw32 ]]; then
+        onnxruntime_dist_name=onnxruntime-win-x64-$onnxruntime_version.zip
+    elif [[ $target == i686-w64-mingw32 ]]; then
+        onnxruntime_dist_name=onnxruntime-win-x86-$onnxruntime_version.zip
+    fi
+
+    wget https://github.com/microsoft/onnxruntime/releases/download/v$onnxruntime_version/$onnxruntime_dist_name
+    unzip $onnxruntime_dist_name
+    mkdir -p \$includedir \$libdir
+    cp -a onnxruntime*/include/* \$includedir
+    chmod 755 onnxruntime*/lib/*
+    cp -a onnxruntime*/lib/* \$libdir
+    install_license onnxruntime*/LICENSE
+else
+    if [[ $target == aarch64-apple-darwin ]]; then
+        CMAKE_EXTRA_ARGS+=-DCMAKE_OSX_ARCHITECTURES=arm64
+    fi
+
+    wget https://github.com/protocolbuffers/protobuf/releases/download/v3.16.1/protoc-3.16.1-linux-x86_64.zip
+    unzip -d protoc protoc-3.16.1-linux-x86_64.zip
+
+    cd onnxruntime
+    git submodule update --init --recursive
+    mkdir -p build
+    cd build
+    cmake $WORKSPACE/srcdir/onnxruntime/cmake \
+        -DCMAKE_INSTALL_PREFIX=$prefix \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DONNX_CUSTOM_PROTOC_EXECUTABLE=$WORKSPACE/srcdir/protoc/bin/protoc \
+        -Donnxruntime_BUILD_SHARED_LIB=ON \
+        -Donnxruntime_BUILD_UNIT_TESTS=OFF \
+        -Donnxruntime_CROSS_COMPILING=ON \
+        $CMAKE_EXTRA_ARGS
+    make -j $nproc
+    make install
+    install_license $WORKSPACE/srcdir/onnxruntime/LICENSE
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+function platform_exclude_filter(p::Platform)
+    libc(p) == "musl" ||
+    p == Platform("i686", "Linux") ||
```

Comment:
What's the problem with this one?

Reply:
Some linker error: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=16872&view=logs&jobId=e2698624-9414-515d-1025-6869b251ec37&j=e2698624-9414-515d-1025-6869b251ec37&t=7b855fa8-72f2-562c-496e-00e91cb30c13

Reply:
It (also) fails to link `libonnxruntime.so` with `onnxruntime_BUILD_UNIT_TESTS=OFF`: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=16907&view=logs&j=e2698624-9414-515d-1025-6869b251ec37&t=7b855fa8-72f2-562c-496e-00e91cb30c13&l=2156